src/valora-dapp-list.json

```diff
@@ -108,14 +108,23 @@
       "listOnIos": true
     },
     {
-      "name": "HaloFi",
-      "id": "halofi",
+      "name": "HaloFi Challenges",
+      "id": "halofi-challenges",
```

Comment:
Can you remove this change to the id for the existing entry; it could cause issues with analytics and dapp ordering.

Reply:
@MuckT,
Of course. PR is updated w/ requested changes and ready for review.